Libplanet/Blockchain/BlockChain.cs

```diff
@@ -967,9 +967,13 @@ public long GetNextTxNonce(Address address)
                 _rwlock.ExitUpgradeableReadLock();
             }
 
-            if (renderActions)
+            if (renderBlocks && ActionRenderers.Any())
             {
-                RenderBlock(evaluations, block, stateCompleters);
+                if (renderActions)
+                {
+                    RenderBlock(evaluations, block, stateCompleters);
```

Comment:
Do not invoke `RenderBlock` when `renderActions` is false but `renderBlocks` is true?

Reply:
Although the name is confusing, `RenderBlock()` here refers to `BlockChain<T>.RenderBlock()` method which calls `IActionRenderer<T>.RenderAction()` etc, and it does not call `IRenderer<T>.RenderBlock()` method.